composer.json

```diff
@@ -17,7 +17,10 @@
         "hhvm/hhast": "^4.0",
         "facebook/definition-finder": "^2.0.0",
         "hhvm/hacktest": "^2.0",
-        "usox/hackmock": "dev-master",
```

Comment:
Not needed if we're using a tagged release of hackttp, because those don't contain tests (dev-master contains tests and I couldn't find a way to remove them).

Having this here is undesirable because then we also have to ensure that hackmock is compatible with the latest HHVM, in addition to hackttp.